docs/blog/posts/using-vega-protocol-with-hummingbot/index.md

```diff
@@ -0,0 +1,172 @@
+
+You're now ready to deposit or transfer assets and use your new Vega keypair.
+
+### Deposit Funds
+
+!!! Note
+    Please make sure you have enough balance in your wallet including ETH for gas fees. 
+
+In order to trade on Vega we'll need to deposit funds first. Click on the Deposit button in the right hand side of the screen
+
+[![image](image5.png)](image5.png)
+
+You may get a message asking you to approve first the use of the chosen asset before you are able to deposit. Make sure to approve this.
+
+[![image](image6.png)](image6.png)
+
+In the Deposit window double check the `Vega Key` then click the Asset drop down box and select the Asset you want to deposit. In the example below we are going to select `USDT Faucet` and enter in `500` under the Amount. Double check the above details again then once satisfied, click Deposit. 
+
+[![image](image7.png)](image7.png)
+
+You'll then need to approve the transaction in Metamask - click "Confirm" to confirm the transaction. 
+
+[![image](image8.png)](image8.png)
+
+Once this is done, you'll just need to wait for the transaction to be confirmed. You should get a confirmation message once this is done. 
+
+[![image](image9.png)](image9.png)
+
+
+## Install Hummingbot
+
+There are two main methods to install Hummingbot:
+
+### Docker
+
+For most new users, we recommend installing Hummingbot using Docker.
+
+- [Docker Quickstart Guide](https://hummingbot.org/academy-content/docker-installation-guide/)
+
+### Source
+
+We recommend installing Hummingbot from source if you meet any of the following criteria:
+
+- You want to customize or extend the Hummingbot codebase.
+- You want to build new components like connectors or strategies.
+- You want to learn how Hummingbot works at a deeper, technical level.
+
+Guides for various environments:
+  - [Linux](../../../installation/linux.md)
+  - [Windows](../../../installation/windows.md)
+  - [MacOS](../../../installation/mac.md)
+
+
+## Connect your Vega Wallet to Hummingbot
+
+In order to connect your Vega Wallet to Hummingbot you'll need your `SNAP KEY` (also known as Vega Party ID) and your `wallet seed phrase`. 
+
+Launch Hummingbot and then enter the command below in the Hummingbot terminal
+
+`connect vega_perpetual`
+
+or if you are connecting to the testnet
+
+`connect vega_perpetual_testnet`
+
+You'll be prompted to enter in your Vega public key (party ID) - this can be found in the top right of the screen where it says "SNAP KEY". 
+
+Next, enter in the wallet seed phrase and if both credentials are valid you should get the message 
+"You are now connected to Vega Perpetual"
+
+[![image](image4.png)](image4.png)
+
+Once connected to Hummingbot, you should then be able to confirm it's connected by running the `balance` command
+
+## Starting Your First Trade
+
+`v2_dman_v3_multiple_exchanges.py`
+
+Due to the unconventional method that Vega displays the tokens (instead of BASE-QUOTE), the existing scripts will need to be modified  
```

Comment:
Since users can't copy from the screenshot, I recommend an alternative approach for this section

1. Describe the difference with Vega tokens and other exchanges
2. Include a Gist with a sample script that shows the change needed

Reply:
Also, it would be good to include a screenshot that shows that the script running.